### PR TITLE
feature: open with in cordova app :busstop:

### PR DESCRIPTION
**Pull request by enguerran**
On mobile device, you can *"Open With..."* a file stored in your cozy.

It downloads the file to a temporary folder then let user opens the file in another application.

open with...

- [x] "Open with..." disney scenario
- [ ] Works on both Android and iOS
- [x] "Open with..." if no application can open this type
- [x] "Open with..." if there is no network
- [x] T9n and i18n
- [x] Add a loader in the menu
- [ ] Publish the android-FIL-7.apk

### CODE REVIEW

**Comment by enguerran** [comment]
Here is how an error is displayed

ezgif com-optimize 1

**Comment by enguerran** [comment]
And now, ladies and gentlemen, please welcome the spinner:

ezgif com-optimize 2

File: src/actions/index.js [hunk 1]
```diff
@@ -160,6 +164,26 @@ export const downloadFile = id => {
   }
 }
 
+export const openFileWith = (id, filename) => {
+  return async (dispatch, getState) => {
+    if (window.cordova && window.cordova.plugins.fileOpener2) {
+      dispatch({ type: OPEN_FILE_WITH, id })
+      const response = await cozy.files.downloadById(id).catch((error) => {
```
Comment (kosssi) [inline src/actions/index.js]: I didn't know It's possible to use await with .catch 👍

**Comment by enguerran** [comment]
I rebase this branch to allow anyone to merge it.
Are we cool like Fonzy?

File: src/components/App.jsx [hunk 2]
```diff
@@ -38,6 +39,7 @@ class App extends Component {
         <main class={styles['fil-content']}>
           <Topbar />
           { children }
+          { toastMessage && <Toast />}
```
Comment (kosssi) [inline src/components/App.jsx]: It's normal to call an container in component?

Comment (enguerran) [inline src/components/App.jsx]: Why not?
Containers and Components are both Components as known as Dumb Components and Smart Components.

What differentiate smart and dumb? Within `redux` and `react-redux`, a Smart Component is a component that fetches data from state or store's dispatch (*actions*), so you could have a Component tree with different layers including SC or DC in any order.

Do you agree @goldoraf?

**Pull request merged**